modules/light-clients/08-wasm/keeper/events.go

```diff
@@ -5,19 +5,19 @@ import (
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
 
-	clienttypes "github.com/cosmos/ibc-go/v7/modules/core/02-client/types"
+	"github.com/cosmos/ibc-go/modules/light-clients/08-wasm/types"
 )
 
 // emitCreateClientEvent emits a create client event
 func emitStoreWasmCodeEvent(ctx sdk.Context, codeHash []byte) {
 	ctx.EventManager().EmitEvents(sdk.Events{
 		sdk.NewEvent(
-			clienttypes.EventTypeStoreWasmCode,
-			sdk.NewAttribute(clienttypes.AttributeKeyWasmCodeHash, hex.EncodeToString(codeHash)),
+			types.EventTypeStoreWasmCode,
+			sdk.NewAttribute(types.AttributeKeyWasmCodeHash, hex.EncodeToString(codeHash)),
 		),
 		sdk.NewEvent(
 			sdk.EventTypeMessage,
```

Comment:
i guess per @colin-axner 's comment in the call, this section might not be necessary, because this is not a submodule?

Reply:
I think we may actually need to keep this. 

It's not necessarily to do with submodules, but a msg's type url. And because `MsgStoreCode` has a type url of [ibc.lightclients.wasm.v1.MsgStoreCode](https://github.com/cosmos/ibc-go/blob/feat/wasm-clients/proto/ibc/lightclients/wasm/v1/tx.proto#L2), when [sdk events are created](https://github.com/cosmos/cosmos-sdk/blob/13c66e12b84c611162041d04a71fc92f9362c9f3/baseapp/baseapp.go#L1049-L1051) - [it will parses index 1](https://github.com/cosmos/cosmos-sdk/blob/main/types/tx_msg.go#L131) from it and then we would end up with module name as `lightclients`.

Reply:
ah yea that's correct